docs/overview/contributing/setting_up.rst

```diff
@@ -239,14 +239,14 @@ This is a builtin package and doesn't require explicit installation.
 
       pip install -r requirements.txt
 
-   This will install packages required for running the tests in Array API suite.
+   This will install packages required for running the tests in the Array API suite.
 
 Here are the visual guides for setting up a `virtualenv environment <https://www.jetbrains.com/help/pycharm/creating-virtual-environment.html#0>`_ OR `conda environment <https://www.jetbrains.com/help/pycharm/conda-support-creating-conda-virtual-environment.html>`_ in pycharm from JetBrains.
 For VSCode, you can follow the instructions `virtual environments <https://code.visualstudio.com/docs/python/environments#_creating-environments>`_.
 
 **Installing Ivy from source**
 
-You can also install Ivy from source if you want to take advantage of the latest changes, but we can't ensure everything will work as expected. All the steps will remain the same for miniconda and venv as described above, only the command for point 4 for venv and point 5 for miniconda will change, everything else will remain the same. You have to run the following instead:
+You can also install Ivy from the source if you want to take advantage of the latest changes, but we can't ensure everything will work as expected. All the steps will remain the same for miniconda and venv as described above, only the command for point 4 for venv and point 5 for miniconda will change, everything else will remain the same. You have to run the following instead:
```

Comment:
Again, I believe install from source is the right wording

Reply:
Here too, same as above.